pkg/pillar/scripts/device-steps.sh

```diff
@@ -202,6 +203,33 @@ if [ -f $CONFIGDIR/hardwaremodel ]; then
     mv $CONFIGDIR/hardwaremodel $PERSISTDIR/status
 fi
 
+# Checking for low diskspace at bootup. If used percentage of
+# /persist directory is more than 70% then we will remove the
+# following sub directories:
+# /persist/log/*
+# /persist/newlog/appUpload/*
+# /persist/newlog/devUpload/*
+# /persist/newlog/keepSentQueue/*
+# /persist/newlog/failedUpload/*
+diskspace_used=$(df /persist |awk '/\/dev\//{printf("%d",$5);}')
+echo "Used percentage of /persist: $diskspace_used"
+if [ $diskspace_used > $DISKSPACE_RECOVERY_LIMIT ]
+then
+    echo "Used percentage of /persist is $diskspace_used more than the limit $DISKSPACE_RECOVERY_LIMIT"
+    for VARIABLE in $PERSISTDIR/log/* $PERSISTDIR/newlog/keepSentQueue/* $PERSISTDIR/newlog/failedUpload/* $PERSISTDIR/newlog/appUpload/* $PERSISTDIR/newlog/devUpload/*
```

Comment:
I think you want the list to be the directories and not their content thus no trailing "*" and then do a rm -rf $dir/* below.

Reply:
Resolved.